src/backend/app/submissions/submission_crud.py

```diff
@@ -358,14 +358,17 @@ def update_submission_in_s3(
         try:
             # Get the last submission date from the metadata
             file = get_obj_from_bucket(settings.S3_BUCKET_NAME, metadata_s3_path)
-            zip_file_last_submission = (json.loads(file.getvalue()))["last_submission"]
-            if last_submission <= zip_file_last_submission:
-                # Update background task status to COMPLETED
-                update_bg_task_sync = async_to_sync(
-                    project_crud.update_background_task_status_in_database
-                )
-                update_bg_task_sync(db, background_task_id, 4)  # 4 is COMPLETED
-                return
+            if file:
```

Comment:
For this kind of check, I always find it good to do a pattern like:

```python
if not file:
    return # note this should probably return something, even if None
# continue logic here
```

it prevents the extra indenting